bubblepicker/src/main/java/com/igalata/bubblepicker/physics/Border.kt

```diff
@@ -31,6 +31,8 @@ class Border(world: World, var position: Vec2, var view: Int) {
             return FixtureDef().apply {
                 this.shape = this@Border.shape
                 density = 50f
+                restitution = 0.99f
```

Comment:
Wouldn't this mean that, eventually, the bubbles would run out of kinetic energy and would stop?